core/utils.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import
+
+from pure_pagination import EmptyPage
+from pure_pagination import Paginator
+from django.shortcuts import render
+
+
+def create_child_instance_from_parent(child_cls, parent_instance):
```

Comment:
Migration would be à pain after all i would rather redactor this and delay the release a bit ill do this tonight why i cover views with functional tests its a major investment for me but hopefully it'll pay off